staging/src/k8s.io/cloud-provider/app/testing/testserver.go

```diff
@@ -109,6 +109,8 @@ func StartTestServer(ctx context.Context, customFlags []string) (result TestServ
 		return TestServer{}, err
 	}
 
+	s.Generic.LeaderElection.LeaderElect = false
```

Comment:
this code was not used, next commit adds an integration test using it, and leader election is not really useful here